Core/clim-core/gadgets.lisp

```diff
@@ -2187,6 +2184,15 @@ response to scroll wheel events.")
   (+ (text-style-ascent  (pane-text-style pane) pane)
      (text-style-descent (pane-text-style pane) pane)))
 
+(defmethod visible-items ((pane generic-list-pane))
+  (if (slot-boundp pane 'visible-items)
+      (visible-items pane)
+      (generic-list-pane-items-length pane)))
+
+(defmethod (setf visible-items) (new-value (pane generic-list-pane))
+  (check-type new-value fixnum)
```

Comment:
what do you mean by accessor arg? I have removed it from class definition. Regarding unbound slot you are right, I'll fix it in the upcoming PR.